clients/src/main/java/org/apache/kafka/common/protocol/types/Type.java

```diff
@@ -427,49 +427,49 @@ public ByteBuffer validate(Object item) {
         }
     };
 
-    public static final Type RECORDS = new Type() {
+    public static final Type LOGBUFFER = new Type() {
```

Comment:
Is it intentional that this is `LOGBUFFER` instead of `LOG_BUFFER`? Same for the `toString` implementation.